packages/@jsii/dotnet-runtime-test/generate.sh

```diff
@@ -1,9 +1,20 @@
 #!/bin/bash
 set -euo pipefail
 test="./test"
+genRoot="${test}/generated"
 
-# Generate NuGet packages for jsii-calc and its dependencies.
-jsii-pacmak -t dotnet --recurse -o ${test}/generated ../../jsii-calc
+# Clean up before we start working
+rm -rf genRoot
```

Comment:
```suggestion
rm -rf ${genRoot}
```

Reply:
Good catch! (I had made this mistake in a bunch of other places, but apparently this one survived the fixing :)